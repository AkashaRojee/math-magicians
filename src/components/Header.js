import WebsiteTitle from './WebsiteTitle';
import NavigationBar from './NavigationBar';
import SplitPane from './SplitPane';

const Header = () => (
  <SplitPane
    left={<WebsiteTitle />}
    right={<NavigationBar />}
  />
);

export default Header;
