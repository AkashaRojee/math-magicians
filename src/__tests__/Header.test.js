import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Header from '../components/Header';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Header />
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
