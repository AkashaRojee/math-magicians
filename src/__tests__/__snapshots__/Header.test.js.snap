// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="header"
>
  <h1>
    Math Magicians
  </h1>
  <nav>
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</div>
`;
